src/java.desktop/share/classes/javax/swing/UIDefaults.java

```diff
@@ -393,7 +398,7 @@ public Object put(Object key, Object value) {
      * @see #put
      * @see java.util.Hashtable#put
      */
-    public void putDefaults(Object[] keyValueList) {
+    public void putDefaults(@Nullable Object[] keyValueList) {
```

Comment:
Here, `@Nullable` is in the right place: on the array element type rather than the array type.

Reply:
This is a correct annotation, right? The array itself cannot be null,  but the objects inside the array can be null.

Reply:
Right.  That is what my comment was trying to say; sorry it was unclear.